kratos/geometries/geometry.h

```diff
@@ -921,6 +921,7 @@ class Geometry : public PointerVector<TPointType>
             auto norm2DXi = norm_2(DeltaXi);
 
             if(norm2DXi > 30 || norm2DXi < tol) {
+                KRATOS_WARNING("Geometry") << "Computation of local coordinates failed at iteration " << k << std::endl;
```

Comment:
this is an error not a warning...

Reply:
If I throw an error cases where the point to compute is very far* will stop the simulation.

* (like in contact, where you may be interested in check this kind of cases)

Reply:
This was the motive why the quadrialteral contact was more robust than the triangular contact.

Reply:
It was throwing an  error only in the triangle 3d, in the other classes was using just a std::cout, I did the change for consistency and in my opinion is the proper way

Reply:
Besides I am using heavily too